static/app/utils/analytics/onboardingAnalyticsEvents.tsx

```diff
@@ -17,6 +17,14 @@ export type OnboardingEventParameters = {
     platform: string;
     project_id: string;
   };
+  'onboarding.js_loader_show_configuration': {
```

Comment:
shall we also track when the page is loaded?

Reply:
I wasn't sure how much I can/should add there. Maybe just add a general tracking to `setupDocs` with the platform & which type of onboarding was used?

Reply:
So I added a new analytic that captures which mode we are rendering, which can be `integration`, `loader`, `react` or `default`. WDYT?

Reply:
hm... integration and default are not clear to me... I would prefer to have only one for this specific case for now... 

I would trigger an analytics code when the page with the loader content is loaded

Reply:
Ok, sure! Can I just put that into the `SetupDocsLoader` function directly then?

Reply:
yes sounds good...ty!

Reply:
👍 updated this!